src/DynamoRevit/DynamoRevit.csproj

```diff
@@ -47,7 +47,12 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DynamoApplications">
-      <HintPath>$(DYNAMOAPI)\DynamoApplications.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\Program Files\Dynamo\Dynamo Core\1.2\DynamoApplications.dll</HintPath>
```

Comment:
I think I restored them all back to what they were. I see now, how you use the $(DYNAMOAPI) variable to set all paths from the user_local.props file. My apologies for messing it up.